### ISSUE TITLE
Cannot install openssl from a machine without internet access

### ISSUE THREAD

**Issue by KineticTheory** [LLNL/spack#844]
I expect to use sources from a local mirror when installing software on machines without internet access.  Spack fails to install openssl on such a system because it requires a ping of the download url.  The error is:

```
% spack install -v openssl%gcc
==> Error: Class constructor failed for package 'openssl'.

Caused by:
IOError: [Errno socket error] [Errno 101] Network is unreachable
```

I can hack the `openssl/package.py` to make it work, but I think there should be a requirement that packages work for offline machines.

Comments?

**Comment by alalazo** [comment]
@KineticTheory Can you check if the issue is resolved by #703 ? If I remember well I modified the package so that your case will just warn the user that an old version is possibly being used...

**Comment by KineticTheory** [comment]
@alalazo Indeed the patch from #703 fixes this issue.  Closing PR #844.
